src/Validation/Validator.php

```diff
@@ -1619,6 +1619,24 @@ public function isArray($field, $message = null, $when = null)
     }
 
     /**
+     * Add a validation rule to ensure that a field contains a scalar.
```

Comment:
Should we add a small note that it means? That it would not be an object or array?